validation/superchain-version_test.go

```diff
@@ -80,69 +57,117 @@ func TestSuperchainWideContractVersions(t *testing.T) {
 
 func TestContractVersions(t *testing.T) {
 	isExcluded := map[uint64]bool{
-		919:       true, // sepolia/mode   L1CrossDomainMessengerProxy.version=1.4.1, https://github.com/ethereum-optimism/security-pod/issues/105
-		1740:      true, // sepolia/metal  L1CrossDomainMessengerProxy.version=1.4.1, https://github.com/ethereum-optimism/security-pod/issues/105
-		1750:      true, // mainnet/metal  L1CrossDomainMessengerProxy.version=1.4.1, https://github.com/ethereum-optimism/security-pod/issues/105
-		8453:      true, // mainnet/base
-		8866:      true, // mainnet/superlumio L1CrossDomainMessengerProxy.version=1.4.1, https://github.com/ethereum-optimism/security-pod/issues/105
-		34443:     true, // mainnet/mode
-		84532:     true, // sepolia/base
-		90001:     true, // sepolia/race, due to https://github.com/ethereum-optimism/superchain-registry/issues/147
-		7777777:   true, // mainnet/zora
-		11155421:  true, // sepolia-dev-0/oplabs-devnet-0
-		999999999: true, // sepolia/zora
-	}
-
-	checkOPChainSatisfiesSemver := func(t *testing.T, chain *ChainConfig) {
-		// TODO: don't hardcode this https://github.com/ethereum-optimism/superchain-registry/issues/219
-		isFaultProofChain := map[uint64]bool{
-			10:       true, // OP Mainnet
-			11155420: true, // OP Sepolia
-		}
+		8453:     true, // mainnet/base
+		11155421: true, // sepolia-dev-0/oplabs-devnet-0
+		11763072: true, // sepolia-dev-0/base-devnet-0
+	}
 
+	checkOPChainMatchesATag := func(t *testing.T, chain *ChainConfig) {
 		rpcEndpoint := Superchains[chain.Superchain].Config.L1.PublicRPC
-
 		require.NotEmpty(t, rpcEndpoint)
 
 		client, err := ethclient.Dial(rpcEndpoint)
 		require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+		require.NotNil(t, chain.ContractsVersionTag, "Chain does not declare a contracts_version_tag")
 
-		contractNames := BaseStackContractNames
-		if isFaultProofChain[chain.ChainID] {
-			// The FaultProof contracts are only present in chains that have activated fault proofs
-			contractNames = append(contractNames, FaultProofContractNames...)
-		} else {
-			// The L2OutputOracle contract is only present in chains that have not activated fault proofs
-			contractNames = append(contractNames, "L2OutputOracle")
-		}
+		isFPAC := *chain.ContractsVersionTag == "op-contracts/v1.4.0"
```

Comment:
In other places you use `isFaultProofs` instead of `isFPAC`. Would be nice to change this as well for consistency (and eliminating the acronym reduces confusion)

Reply:
Yes already filed an issue for this, will follow-up shortly https://github.com/ethereum-optimism/superchain-registry/issues/301